spotlessSelf.gradle

```diff
@@ -24,9 +24,12 @@ spotless {
 		trimTrailingWhitespace()
 	}
 	groovy {
-		target 'build.gradle', 'spotlessSelf.gradle', 'settings.gradle'
-		greclipseFormat().configFile('spotless.eclipseformat.xml')
-		paddedCell()
+		target fileTree('.') {
+			include '**/*.gradle'
+			exclude '_ext/**'
+		}
+		custom 'preventFormatPingPong', { return it.replaceAll('}[ \t\f]+}', '}}') }
```

Comment:
I'm mildly curious by the `\f` here, as it's a unusual regex pattern for me. What does it match against?

Would matching against `\t` as well be sensible here?